hardware/opentrons_hardware/sensors/mmr920C04.py

```diff
@@ -26,6 +46,77 @@ def __init__(
         """Constructor."""
         super().__init__(zero_threshold, stop_threshold, offset, SensorType.pressure)
 
+    async def bind_to_sync(
+        self,
+        can_messenger: CanMessenger,
+        node_id: NodeId,
+        timeout: int = 1,
+    ) -> None:
+        """Send a BindSensorOutputResponse."""
+        await can_messenger.send(
+            node_id=node_id,
+            message=BindSensorOutputRequest(
+                payload=BindSensorOutputRequestPayload(
+                    sensor=SensorTypeField(SensorType.capacitive),
```

Comment:
```suggestion
                    sensor=SensorTypeField(SensorType.pressure),
```
?

In general, might be a good idea to put this in something reusable